### ISSUE TITLE
Dimension of Softmax is along the Digits

### ISSUE THREAD

**Issue by saswatpp** [jindongwang/Pytorch-CapsuleNet#8]
https://github.com/jindongwang/Pytorch-CapsuleNet/blob/93996119a224e9c3d274df88735f1acb36592d97/capsnet.py#L101

here the dim parameter should be set to dim=1 (softmax has to be calculated along the digits axis )
